lib/howitzer/meta/section.rb

```diff
@@ -0,0 +1,43 @@
+module Howitzer
+  module Meta
+    # This class represents section entity within howitzer meta information interface
+    class Section
+      attr_reader :name, :context
+
+      def initialize(name, context)
+        @name = name
+        @context = context
+      end
+
+      # Finds all instances of section on the page and returns them as array of capybara elements
+      # @return [Array]
+      def capybara_elements
+        context.send("#{name}_sections").map(&:capybara_context)
+      end
+
+      # Finds section on the page and returns as a capybara element
+      # @return [Capybara::Node::Element, nil]
+      def capybara_element
+        section = context.send("#{name}_sections").first
+        section.nil? ? nil : section.capybara_context
+      end
+
+      # Highlights element with red border on the page
+      def highlight
```

Comment:
This method is totally the same like another `highlight` methods described above
let's extract these methods to some service object or some module and include it into classes where this method described.